### ISSUE TITLE
Underlines missing on links

### ISSUE THREAD

**Issue by aardrian** [tc39/tc39.github.io#138]
The links for the feature list items are distinguished by color alone (see [WCAG 2.1 SC 1.4.1](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141#use-of-color) and [F73: Failure of 1.4.1](https://www.w3.org/TR/WCAG20-TECHS/F73.html)). Users with low vision may only know these are links by hovering them with a pointer, keyboard-only users can only tell by tabbing to them, and touch users cannot know without tapping.

The quickest solution is to remove this CSS declaration from style.css:

`.featurelist__item h4 a {
    text-decoration: none;
}`

Without wading into all the CSS and/or all pages of the site, I encourage you to find all instances of `text-decoration: none;` and remove it.

**Comment by aardrian** [comment]
Fixed with https://github.com/tc39/tc39.github.io/pull/141.
